packages/config/config/devices/0x0113/lrm-as.json

```diff
@@ -36,18 +44,62 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No",
+					"label": "Use Start Level",
 					"value": 0
 				},
 				{
-					"label": "Yes",
+					"label": "Ignore Start Level",
```

Comment:
This should be reworded so the options can be enable/disable per the style guide.